app/services/concerns/has_diversity_attributes.rb

```diff
@@ -1,83 +1,58 @@
 # frozen_string_literal: true
 
-module DiversityAttributes
```

Comment:
The diff here looks confusing.

But this concern was mostly just being used in the `CreateFromDttp` service. `CreateFromHesa` was already overwriting most of these methods since the two services were starting to diverge.

What I did here was:
- Move all the DTTP-specific code into the DTTP service
- Copy all the overwriting methods from the HESA service into here
- Use this concern in HESA and HPITT services, rather than HESA and DTTP